roles/openshift_service_catalog/tasks/wire_aggregator.yml

```diff
@@ -138,6 +143,10 @@
       value: [X-Remote-Group]
     - key: authConfig.requestHeader.extraHeaderPrefixes
       value: [X-Remote-Extra-]
+    - key: assetConfig.extensionScripts
+      value: [/etc/origin/master/openshift-ansible-catalog-console.js]
+    - key: kubernetesMasterConfig.apiServerArguments.runtime-config
+      value: [apis/settings.k8s.io/v1alpha1=true]
```

Comment:
its to ensure that its a list, if it isn't added as a list item the master will fail to start up